v23.1/tsquery.md

```diff
@@ -0,0 +1,63 @@
+---
+title: TSQUERY
+summary: The TSQUERY data type a list of lexemes separated by operators, and is used in full-text search.
+toc: true
+docs_area: reference.sql
+---
+
+The `TSQUERY` [data type](data-types.html) stores a list of lexemes separated by operators. `TSQUERY` values are used in [full-text search](full-text-search.html).
+
+## Syntax
+
+A `TSQUERY` comprises individual lexemes and operators in the form: `'These' & 'lexemes' & 'are' & 'not' & 'normalized.'`.
+
+The operators in a `TSQUERY` are used to [match a `TSQUERY` to a `TSVECTOR`](full-text-search.html#match-queries-to-documents). Valid `TSQUERY` operators are:
+
+- `&` (AND)
+- `|` (OR)
+- `!` (NOT)
+- `<->` (FOLLOWED BY)
+
+You can optionally add the following to each lexeme:
+
+- One or more weight letters (`A`, `B`, `C`, or `D`): 
```

Comment:
Let's at least explain what the weights are for - weights in queries match the weights in vectors. If you haven't explicitly specified weights in vectors, no point in specifying them in queries. Additionally, query lexemes can have more than 1 weight attached (indicating any will match) whereas vector lexemes can just have 1.